src/firefly/js/visualize/task/PlotChangeTask.js

```diff
@@ -64,7 +66,17 @@ const dispatchAndMaybeMatch= (rawAction) => (dispatcher,getState) => {
     };
 
 
-export const recenterActionCreator= (rawAction) => dispatchAndMaybeMatch(rawAction);
+export const recenterActionCreator = (rawAction) => {
+    // `recenter` reducer expects centerPt to be WorldPtObj
+    const {centerPt} = rawAction.payload;
+    if (centerPt?.type === Point.W_PT && (!centerPt?.cSys || isString(centerPt?.cSys))) {
+        // type is WorldPt, and cSys is str or undefined (aka is not CoordinateSys object)
+        rawAction.payload.centerPt = makeWorldPt(centerPt.x, centerPt.y, CoordSys.parse(centerPt.cSys));
```

Comment:
Firefly currently support two formats for work point.
- The world point object
- The serialized version `'ra;dec;coords'` for example `'1.111;2.222;EQ_J2000'`

You are actually proposing the third format a json object with cSys a string and not a constant that we then parse to a world point object. 

I think that is ok but this is not the place for it. It needs to be consistent across firefly.  It would be better for you to call `parseWorldPt` in `Point.js` and enhance it to support this format. In that case `parseWorldPt` should do all the work.

Reply:
I realised the serialsed version can do the same job if I use `parseAnyPt`. This even allows passing not WorlPt points like "SCREEN_PIXEL" etc. Thanks for the heads up!

Reply:
For testing: check if following works correctly:
- recentre option in image toolbar 
- existing method in notebook `fc.set_pan` with coord=
  - 'j2000'
  - 'image'
  - 'SCREEN_PIXEL'